documentation/contracts/bot-out.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://salz.life/bot-out.schema.json",
+  "title": "BotOutMessage",
+  "description": "A message from the bot to the match handler.",
+  "type": "object",
+  "oneOf": [
+    {
+      "properties": {
+        "messageType": { "const": "debugMessage" },
+        "body": { "type": "string"}
+      }
+    },
+    {
+      "properties": {
+        "messageType": { "const": "errorMessage" },
+        "body": { "type": "string"}
+      }
+    },
+    {
+      "properties": {
+        "messageType": { "const": "gameAction" },
+        "body": { "type": "object"}
+      }
+    }
```

Comment:
Should we put that `debugMessage` into the `gameAction` type so that debug messages can be shared even on a successful game action?